app.py

```diff
@@ -627,8 +629,14 @@ def edit():
 		tree, senttok = discbrackettree(request.args.get('tree'))
 	else:
 		return 'ERROR: pass n or tree argument.'
-	treestr = writediscbrackettree(tree, senttok, pretty=True).rstrip()
-
+	if app.config['CGELVALIDATE'] is None:
+		treestr = writediscbrackettree(tree, senttok, pretty=True).rstrip()
+		rows = max(5, treestr.count('\n') + 1)
+	else: 
+		block = writetree(tree, senttok, '1', 'export', comment='')  #comment='%s %r' % (username, actions))
```

Comment:
What is `'1'` for?

Reply:
`'1'` is a (trivial/meaningless) value for `writetree`'s `key` argument (see https://github.com/andreasvc/disco-dop/blob/dd181a6633e7373eed3f7049537d30e79c7ffa56/discodop/treebank.py#L705)

Reply:
Can it be `None`? Or if it needs to be an arbitrary string I would leave a comment to that effect.

Reply:
It looks like it needs to be an arbitrary string, so I've left a comment where this occurs